overcooked_ai_py/mdp/overcooked_mdp.py

```diff
@@ -1174,6 +1196,172 @@ def get_deltas_to_closest_location(self, player, locations, mlp):
     # DEPRECATED #
     ##############
 
+    """---Added by pk: But these aren't needed AND seem to use an old version of the state encoding, 
+    because sometimes state.objects == state_from_obs_from_state.objects...???"""
+    # def check_state_to_obs_to_state(self, observation, player_observing, state):
+    #
+    #     state_from_obs_from_state = self.state_from_observation(observation, player_observing)
+    #     # print(state_from_obs_from_state)
```

Comment:
Remove all of these commented lines. Maybe keep a copy in a local `old` folder, but .gitignore it out of the actual commit.

Did you mean to say  `state.objects != state_from_obs_from_state.objects`? That's not a very good sign, but yeah it is probably due to minor modifications made to the featurization function, and fortunately we don't need that function anymore.